tpp-app/src/settings/SettingsService.js

```diff
@@ -0,0 +1,116 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { TRACK_SYMPTOMS, REMINDERS } from './utils/constants.js'
+
+    /**
+     * Clears all of the user's account data
+     * @returns a promise resolving when the clear operation is complete
+     */
+
+    export const DELETEAccountData = async () => new Promise( async (resolve, reject) => {
+        try {
+            await AsyncStorage.clear().then(() => {
+                console.log("Deleted user account data");
+                resolve(); 
+            })
+            
+        } catch (e) {
+            console.log(`DELETEAccountData error: ${JSON.stringify(e)}`);
+            reject();
+        }
+    })
+
+    /**
+     * Returns an array of a boolean representing if a symptom is currently tracked
+     * @returns a promise of an array containing booleans representing whether to track each symptom
+     */
+    export const GETAllTrackingPreferences = async () => new Promise (async (resolve, reject) => {
+        try {
+             await AsyncStorage.multiGet([
+                TRACK_SYMPTOMS.FLOW,
+                TRACK_SYMPTOMS.MOOD,
+                TRACK_SYMPTOMS.SLEEP,
+                TRACK_SYMPTOMS.CRAMPS,
+                TRACK_SYMPTOMS.EXERCISE
+             ]).then(() => {
+                 console.log("Got All Tracking Preferences");
+                 resolve();
+             }).catch((e) => {
+                console.log(JSON.stringify(e));
+                reject('Unable to getItem');
+             });
+        } catch (e) {
+            console.log(`GETAllTrackingPreferences error: ${JSON.stringify(e)}`)
+            reject();
+        }
+    })
+
+
+    /**
+     * Posts any changes in symptom tracking preferences 
+     * @param flow boolean representing whether to track flow
+     * @param mood boolean representing whether to track mood 
+     * @param sleep boolean representing whether to track sleep 
+     * @param cramps boolean representing whether to track cramps
+     * @param exercise boolean representing whether to track exercise
+     * @returns a promise resolving when the post operation is complete
+     */
+    export const POSTUpdatePreferences = async (flow, mood, sleep, cramps, exercise) => new Promise (async (resolve, reject) => {
```

Comment:
this seems super similar to what i implemented in `POSTSymptomsToTrack` for onboarding services other than that this function sets the tracking preferences as an array? i'm not sure which one we should keep but i feel like we should not need to keep both? i actually prefer this way of storing it all in one array, but i would recommend adding an `.any()` check here to make sure that at least one of the symptoms get tracked